packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -232,10 +238,25 @@ class Dismissible extends StatefulWidget {
   /// This defaults to [HitTestBehavior.opaque].
   final HitTestBehavior behavior;
 
+  /// Called when the dismissible widget has been dragged.
```

Comment:
```suggestion
  /// Called when the dismissible widget has been dragged.
  ///
  /// If [onDismissUpdate] is not null, then it will be invoked for every pointer event
  /// to dispatch the latest state of the drag.
```

Also, an example that expresses why this callback is useful and how to use it would be preferable.

Reply:
I added a sentence to describe how this callback might be useful.